packages/config/config/devices/0x000c/hs-ms100.json

```diff
@@ -24,6 +24,7 @@
 			"minValue": 1,
 			"maxValue": 8,
 			"defaultValue": 8,
+			"unsigned": true,
```

Comment:
That's not necessary when you don't leave the positive signed range (0-127 for 1 byte)